source/features/index.tsx

```diff
@@ -45,32 +45,40 @@ interface InternalRunConfig {
 }
 
 let log: typeof console.log;
+const {version} = browser.runtime.getManifest();
 
-function logError(id: FeatureID, error: Error | string | unknown, ...extras: unknown[]): void {
-	if (error instanceof TypeError && error.message === 'Object(...)(...) is null') {
-		error.message = 'The element wasn’t found, the selector needs to be updated.';
-	}
-
+let logError = (id: FeatureID, error: unknown): void => {
 	const message = error instanceof Error ? error.message : String(error);
 
 	if (message.includes('token')) {
-		console.log(`ℹ️ Refined GitHub → ${id} →`, message);
+		console.log(`ℹ️ ${id} →`, message);
 		return;
 	}
 
-	// Don't change this to `throw Error` because Firefox doesn't show extensions' errors in the console.
-	// Use `return` after calling this function.
-	console.error(
-		`❌ Refined GitHub → ${id} →`,
-		error,
-		...extras,
-		stripIndent(`
-			Search issue: https://github.com/sindresorhus/refined-github/issues?q=is%3Aissue+${encodeURIComponent(message)}
-
-			Open an issue: https://github.com/sindresorhus/refined-github/issues/new?labels=bug&template=bug_report.md&title=${encodeURIComponent(`\`${id}\`: ${message}`)}
-		`)
-	);
-}
+	// Don't change this to `throw Error` because Firefox doesn't show extensions' errors in the console
+	console.group('❌', id, version, pageDetect.isEnterprise() ? 'GHE →' : '→', error);
```

Comment:
# Better layout for errors

- no repeated "Refined GitHub" text
- no repeated stack
- includes RG version
- includes "GHE" if it's on GHE

<img width="543" alt="Screen Shot 2" src="https://user-images.githubusercontent.com/1402241/116373582-ae8bd780-a837-11eb-8680-310178e6bf62.png">